packages/@uppy/utils/package.json

```diff
@@ -59,7 +59,8 @@
     "./lib/delay": "./lib/delay.js",
     "./lib/hasProperty": "./lib/hasProperty.js",
     "./lib/mimeTypes": "./lib/mimeTypes.js",
-    "./lib/getDroppedFiles": "./lib/getDroppedFiles/index.js"
+    "./lib/getDroppedFiles": "./lib/getDroppedFiles/index.js",
+    "./src/microtip.scss": "./src/microtip.scss"
```

Comment:
It's the only export that uses `/src/`, not great for consistency (but great for backward compatibility so..)

Reply:
It's because we import this file in scss, we never compile or want it to be compiled I think